.github/workflows/dev-build.yml

```diff
@@ -0,0 +1,62 @@
+name: Development build
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  dev-build:
+    if: github.event.review.state == 'approved'
+    runs-on: ubuntu-latest
+    name: Development build after approving review
+    steps:
+    - name: Checkout from GIT
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # We later want to describe the parent's hash.
+        submodules: recursive
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
+        extensions: mbstring, mysqli, intl, curl  # optional, setup extensions
+        coverage: xdebug  # optional, setup coverage driver
+
+    - name: Check PHP Version
+      run: php -v
+
+    - name: Composer install
+      run: composer install
+
+    - name: Build for development
+      run: PREFIX=/tmp/release make test-zip
+
+    - name: Get version name
+      # This runs on a merge commit, but we want the commit before the review.
+      # A merge commit has two parents and the one from the PR seems to be at HEAD^2.
+      run: echo "DEV_BUILD_VERSION=$(git describe --tags HEAD^2)" >> $GITHUB_ENV  
+
+    - name: Set version in plugin
+      run: sed -i -r "s/Version:\ [^$]+$/Version:\ ${{ env.DEV_BUILD_VERSION }}/" /tmp/release/onoffice/plugin.php
+
+    - run: cat /tmp/release/onoffice/plugin.php
+
+    - name: Zip files
+      run: (cd /tmp/release/ && zip -r ./onoffice-${{ env.DEV_BUILD_VERSION}}.zip ./onoffice/)  # To prevent nested paths (like /tmp/release/onoffice/readme.txt) we have to cd into the directory.
```

Comment:
I think `/tmp/` should be cleaned up at the end of this action, just to make sure nothing gets cached automatically